src/Symfony/Component/Notifier/Bridge/JoliNotif/JoliNotifTransportFactory.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\JoliNotif;
+
+use Joli\JoliNotif\DefaultNotifier as JoliNotifier;
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+
+/**
+ * @author Ahmed Ghanem <ahmedghanem7361@gmail.com>
+ */
+final class JoliNotifTransportFactory extends AbstractTransportFactory
+{
+    private const SCHEME_NAME = 'jolinotif';
```

Comment:
should the scheme be `joli-notif` to be consistent with the package name ?

Reply:
I feel it's more readable without the dash, I also have seen many other bridges follow the same pattern in naming the scheme without applying the kebab-case.